### PR TITLE
 kernel: make kernel option arch-dependent and update it to v4.19.8 for aarch64

### PR DESCRIPTION
**Pull request by Pennyzct**
For including dev-kernel build process, we should add related options
in version.yaml

Fixes: #972

Signed-off-by: Penny Zheng <penny.zheng@arm.com>
Signed-off-by: Wei Chen <Wei.Chen@arm.com>

### CODE REVIEW

**Comment by Pennyzct** [comment]
This is related to PR [#269](https://github.com/kata-containers/packaging/pull/269) on packaging repo.

File: versions.yaml [hunk 1]
```diff
@@ -109,6 +109,10 @@ assets:
     description: "Linux kernel optimised for virtual machines"
     url: "https://cdn.kernel.org/pub/linux/kernel/v4.x/"
     version: "v4.14.67"
+    dev:
+      description: "dev-kernel for virtual machines"
+      url: "https://git.kernel.org/pub/scm/linux/kernel/git/torvalds/linux.git/snapshot/"
```
Comment (devimc) [inline versions.yaml]: I copied and pasted this url in the browser, I got next message:

`No snapshot name specified`

is this expected?

Comment (Pennyzct) [reply versions.yaml]: my fault. `/snapshot` should be removed.
But since the `rc` version is unacceptable, i need to drop this solution.

**Comment by Pennyzct** [comment]
please review. I dropped the `dev` solution and updated kernel to v4.19.6. all nvdimm-related patches are been backported on v4.19.6.

**Comment by jodh-intel** [comment]
lgtm

/retest

**Comment by jodh-intel** [comment]
Related to #1111.

/cc @grahamwhaley.

**Comment by raravena80** [comment]
@Pennyzct ping, any updates? Thx!

**Comment by grahamwhaley** [comment]
I think, as we have discussed, maybe this is a necessary (and good) change, to allow kernel versions to be potentially architecture dependant.
Right now this will clash with the move to 4.19.20 on #1111 . I'm going to see if we can get #1111 landed, and then see if we still need this PR, and if so, then we rebase and try again...

**Comment by jodh-intel** [comment]
https://github.com/kata-containers/runtime/pull/1111 has now landed and this PR needs an update (conflict).

**Pull request closed without merging**